python/dlr/api.py

```diff
@@ -26,27 +26,44 @@ def get_output_names(self):
     def run(self, input_data):
         raise NotImplementedError
 
+
+def _find_model_file(model_path, ext):
+    if os.path.isfile(model_path) and model_path.endswith(ext):
+        return model_path
+    model_files = glob.glob(os.path.abspath(os.path.join(model_path, ext)))
+    if len(model_files) > 1:
+        raise ValueError('Multiple {} files found under {}'.format(ext, mdel_path))
+    elif len(model_files) == 1:
+        return model_files[0]
+    return None
+
+
 # Wrapper class
 class DLRModel(IDLRModel):
-    def __init__(self, model_path, dev_type='cpu', dev_id=0):
-        #Determine if 3rdparty package needed
-        tf_model_path = None
-        if os.path.isfile(model_path):    
-            if model_path.endswith('.pb'):
-                tf_model_path = model_path
-        else:
-            model_files = glob.glob(os.path.abspath(os.path.join(model_path, '*.pb')))
-            if len(model_files) > 1:
-                raise ValueError('Multiple .pb files found under ' + model_path)
-            elif len(model_files) == 1:
-                tf_model_path = model_files[0]
-        # Default to DLR model
+    def __init__(self, model_path, dev_type=None, dev_id=None):
+        # Find correct runtime implementation for the model
+        tf_model_path = _find_model_file(model_path, '.pb')
+        tflite_model_path = _find_model_file(model_path, '.tflite')
+        # Check if found both Tensorflow and TFLite files
+        if tf_model_path is not None and tflite_model_path is not None:
+            raise ValueError('Found both .pb and .tflite files under {}'.format(mdel_path))
+        # Tensorflow
         if tf_model_path is not None:
             from .tf_model import TFModelImpl
             self._impl = TFModelImpl(tf_model_path, dev_type, dev_id)
-        else:
-            from .dlr_model import DLRModelImpl
-            self._impl = DLRModelImpl(model_path, dev_type, dev_id) 
+            return
+        # TFLite
+        if tflite_model_path is not None:
+            from .tflite_model import TFLiteModelImpl
+            self._impl = TFLiteModelImpl(tflite_model_path, dev_type, dev_id)
+            return
```

Comment:
Is it possible that both of them are `None`? If so, we may need to throw an error.

Reply:
if both TF and TFLite files are not found then the dispatcher will use DLRModelImpl by default